gateway/api_definition_test.go

```diff
@@ -15,15 +15,13 @@ import (
 	textTemplate "text/template"
 	"time"
 
-	"github.com/TykTechnologies/tyk/apidef/oas"
-
-	"github.com/TykTechnologies/storage/persistent/model"
-	"github.com/TykTechnologies/tyk/config"
-	"github.com/TykTechnologies/tyk/rpc"
-
 	"github.com/stretchr/testify/assert"
 
+	persistentModel "github.com/TykTechnologies/storage/persistent/model"
```

Comment:
this could be all in lowercase letters, right?

Reply:
we don't have a convention other than the established one, it's mostly this :)